fastlane/Appfile

```diff
@@ -0,0 +1,6 @@
+# app_identifier("[[APP_IDENTIFIER]]") # The bundle identifier of your app
```

Comment:
Do we even need this file? I think we can remove it completely

Reply:
Removed.